src/banana/data/genpdf/export.py

```diff
@@ -61,7 +61,10 @@ def dump_blocks(name, member, blocks):
     target = pathlib.Path(name) / ("%s_%04d.dat" % (name, member))
     target.parent.mkdir(exist_ok=True)
     with open(target, "w") as o:
-        o.write("PdfType: central\nFormat: lhagrid1\n---\n")
+        if member == 0:
+            o.write("PdfType: central\nFormat: lhagrid1\n---\n")
+        else:
+            o.write("PdfType: replica\nFormat: lhagrid1\n---\n")
```

Comment:
this should be inherited from parent if possible! check this with MSTW and add also a fake NNPDF and test with that one (you can e.g. simply check the first line in that file ...)

Reply:
Yes but my problem is that in this function we do not have the information on whether we are generating from a parent or not (so if we are using toy for instance). So my idea was to put another arg to the dump_blocks function which tells whether to write "central" or something else. However I don't like so much this idea :(

Reply:
why not? just add an other argument (and as said you can use as a fallback `replica`) - otherwise this will simpy be wrong ...

Reply:
ok then I'll do like this :)